recipes/wrfplot/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "wrfplot" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/wrfplot-{{ version }}.tar.gz
+  sha256: 0078422303a835f68299dae893ab6ddfc63b576ac750210a43d95aea4334208c
+
+build:
+  entry_points:
+    - wrfplot=wrfplot.wrfplot:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - setuptools
+    - python >3.8
```

Metadata says it is:

```suggestion
    - python >=3.7
```